flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/source/ValuesDataSourceHelper.java

```diff
@@ -488,4 +494,101 @@ public static List<List<Event>> multiSplitsSingleTable() {
+                        TABLE_1,
+                        generator.generate(
+                                new Object[] {
+                                    BinaryStringData.fromString("1"),
+                                    BinaryStringData.fromString("1")
+                                }));
+        split1.add(insertEvent1);
+        DataChangeEvent insertEvent2 =
+                DataChangeEvent.insertEvent(
+                        TABLE_1,
+                        generator.generate(
+                                new Object[] {
+                                    BinaryStringData.fromString("2"),
+                                    BinaryStringData.fromString("2")
+                                }));
+        split1.add(insertEvent2);
+        DataChangeEvent insertEvent3 =
+                DataChangeEvent.insertEvent(
+                        TABLE_1,
+                        generator.generate(
+                                new Object[] {
+                                    BinaryStringData.fromString("3"),
+                                    BinaryStringData.fromString("3")
+                                }));
+        split1.add(insertEvent3);
+
+        // add column
+        AddColumnEvent.ColumnWithPosition columnWithPosition =
+                new AddColumnEvent.ColumnWithPosition(
+                        Column.physicalColumn("col3", DataTypes.STRING()));
+        AddColumnEvent addColumnEvent =
+                new AddColumnEvent(TABLE_1, Collections.singletonList(columnWithPosition));
+        split1.add(addColumnEvent);
+
+        // rename column
+        /*Map<String, String> nameMapping = new HashMap<>();
+        nameMapping.put("col2", "newCol2");
+        nameMapping.put("col3", "newCol3");
+        RenameColumnEvent renameColumnEvent = new RenameColumnEvent(TABLE_1, nameMapping);
+        split1.add(renameColumnEvent);*/
+
+        // drop column
+        /*DropColumnEvent dropColumnEvent =
+            new DropColumnEvent(
+                TABLE_1,
+                Collections.singletonList(
+                    Column.physicalColumn("newCol2", DataTypes.STRING())));
+        split1.add(dropColumnEvent);*/
+
+        // delete
+        /*split1.add(
+        DataChangeEvent.deleteEvent(
+            TABLE_1,
+            generator.generate(
+                new Object[] {
+                    BinaryStringData.fromString("1"),
+                    BinaryStringData.fromString("1")
+                })));*/
+
+        // update
+        /*split1.add(
+        DataChangeEvent.updateEvent(
+            TABLE_1,
+            generator.generate(
+                new Object[] {
+                    BinaryStringData.fromString("2"),
+                    BinaryStringData.fromString("")
+                }),
+            generator.generate(
+                new Object[] {
+                    BinaryStringData.fromString("2"),
+                    BinaryStringData.fromString("x")
+                })));*/
```

Comment:
Dead code

Reply:
I want to add unit tests for various scenarios, so this code has not been deleted. Next, I will add unit tests over the weekend.